src/3-Practice/02/assets/stylesheets/object/projects/_index.css

```
@@ -0,0 +1,71 @@
+.p-index {
+  width: 100%;
+}
+
+.p-header {
+  width: 100%;
+}
+
+@media screen and (min-width: 796px) {
+  .p-header {
+    padding: 10px 20px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .p-header__image_wrap {
+    max-width: 50px;
```

I think this is better.
```suggestion
    width: 50px;
```

---

修正完了しました。